### PR TITLE
[NEW] banking_mandate_by_email: Send Banking Mandates by e-mail.

### PR DESCRIPTION
**Pull request by angelgarciadelachica**
[T-5734]
@ValentinVinagre

### CODE REVIEW

File: banking_mandate_by_email/README.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+~~~~~~~~~~~~
+
+* Ángel García de la Chica Herrera <angel.garcia@sygel.es>
+* Valentin Vinagre <valentin.vinagre@qubiq.es>
+* Valentin Vinagre <valentin.vinagre@sygel.es>
+
+Maintainer
+~~~~~~~~~~
+
+This module is maintained by the Qubiq.
```
Comment (ValentinVinagre) [inline banking_mandate_by_email/README.rst]: en esta parte cámbialo por sygel, ya que no lo mantienen o al menos no abierto. 👍🏻 .

File: banking_mandate_by_email/README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+#. Go to the command you want to send by email.
+#. Click on the 'SEND BY E-MAIL' button to send the mandate file. 
+
+If you want to send multiple mandates at the same time:
+
+#. You must select them in the tree view -> Action -> Send mail mandate.
+
+
+Bug Tracker
```
Comment (ValentinVinagre) [inline banking_mandate_by_email/README.rst:37]: en esta parte cámbialo por sygel, ya que no lo mantienen o al menos no abierto. 👍🏻 .

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]
algunos cambios

**Comment by angelgarciadelachica** [comment]
@ValentinVinagre Done!

**Comment by HaraldPanten** [comment]
@ValentinVinagre @angelgarciadelachica I think this module could be replaced by this improvement --> https://github.com/OCA/bank-payment/pull/1244

**Comment by HaraldPanten** [comment]
On the other hand, why do you set [NEW] instead of [ADD] as the title of his PR and the commit message?

**Comment by ValentinVinagre** [comment]
> nk this module could be replaced by this improv

pues si, haremos el cambio entonces 👍🏻

**Comment by ValentinVinagre** [comment]
Se cancela debido a que está incorporado en OCA

**Comment by HaraldPanten** [comment]
> Se cancela debido a que está incorporado en OCA

@ValentinVinagre You need to check if the [IMP] commit is included in V15, as I'm not sure about that.

**Pull request closed without merging**